src/backend/marsha/bbb/api.py

```diff
@@ -74,7 +74,10 @@ def get_permissions(self):
                         | core_permissions.IsTokenAdmin
                     )
                 )
-                | IsClassroomPlaylistOrOrganizationAdmin
+                | (
+                    core_permissions.IsParamsPlaylistAdmin
+                    | core_permissions.IsParamsPlaylistAdminThroughOrganization
+                )
```

Comment:
Actually, even if this does not create a bug now, I think we should test the JWT type to assert the `request.user.id` really designate a user (even if this is highly improbable we get a user with the same UID as a resource)..
```suggestion
                | (
                    UserIsAuthenticated  # asserts request.resource is None
                    & (
                        core_permissions.IsParamsPlaylistAdmin
                        | core_permissions.IsParamsPlaylistAdminThroughOrganization
                    )
                )
```
What do you think?

Reply:
Yes I agree, I think it's better to add it and it can be done in the PR IMO.

Reply:
Done